### PR TITLE
Fix tabbed code in PHP page

### PR DESCRIPTION
**Pull request by chalin**
- Followup to #2654
- **Preview**: https://deploy-preview-2656--opentelemetry.netlify.app/docs/instrumentation/php/automatic/#installation

### Screenshots

Before:

> image

After:

> image

### CODE REVIEW

File: content/en/docs/instrumentation/php/automatic.md [hunk 1]
```diff
@@ -55,35 +55,33 @@ The extension can be installed via pecl,
 1. Setup development environment Installing from source requires proper
```
Comment (brettmc) [inline content/en/docs/instrumentation/php/automatic.md:55]:
```suggestion
1. Setup development environment

Installing from source requires proper
```

File: content/en/docs/instrumentation/php/automatic.md [hunk 2]
```diff
@@ -55,35 +55,33 @@ The extension can be installed via pecl,
...
    {{< tabpane lang=shell persistLang=false >}}
 
    {{< tab "Linux (apt)" >}}sudo apt-get install gcc make autoconf{{< /tab >}}
 
    {{< tab "MacOS (homebrew)" >}}brew install gcc make autoconf{{< /tab >}}
 
    {{< /tabpane >}}
-     <!-- prettier-ignore-end -->
 
 2. Build/install the extension With your environment set up you can install the
```
Comment (brettmc) [inline content/en/docs/instrumentation/php/automatic.md:66]:
```suggestion
2. Build/install the extension

With your environment set up you can install the
```

**Review by brettmc — COMMENTED** [review]
I think prettify must have joined the header and first line...

File: content/en/docs/instrumentation/php/automatic.md [hunk 3]
```diff
@@ -55,35 +55,33 @@ The extension can be installed via pecl,
...
    extension:
```
Comment (brettmc) [inline content/en/docs/instrumentation/php/automatic.md:67]:
```suggestion
2. Build/install the extension

With your environment set up you can install the extension:
```

File: content/en/docs/instrumentation/php/automatic.md [hunk 4]
```diff
@@ -55,35 +55,33 @@ The extension can be installed via pecl,
 1. Setup development environment Installing from source requires proper
    development environment and some dependencies:
```
Comment (brettmc) [inline content/en/docs/instrumentation/php/automatic.md:56]:
```suggestion
1. Setup development environment

Installing from source requires proper development environment and
some dependencies:
```

**Comment by chalin** [comment]
@brettmc - thanks for spotting that. I propose that we fix it by ending the step's leading sentence with a period, and let the rest of the text flow from there. I've committed this change. PTAL

**Review by brettmc — APPROVED** [review]

**Review by svrnm — APPROVED** [review]

**Pull request merged**